### PR TITLE
Make the scripts compatible with py3

### PR DESCRIPTION
**Pull request by magopian**
This is a very straightforward change involving mainly adding parens to the print statements.

The non-trivial changes are in openfisca_france/scripts/parameters/baremes_ipp/xls_to_yaml_raw.py and openfisca_france/scripts/parameters/baremes_ipp/yaml_raw_to_yaml_clean.py:
changing `ur` to `r` in regexes.

I'm really baffled that those tests [would pass without failing in py3 on circleCI previously](https://circleci.com/gh/openfisca/openfisca-france/2338#config/containers/0) ...

### CODE REVIEW

**Comment by magopian** [comment]
This is now ready to review <3

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+### 22.3.1 [#1053](https://github.com/openfisca/openfisca-france/pull/1053)
+
+* Amélioration technique.
+* Détails :
+  - Rends tous les scripts compatibles avec Python 3
```
Comment (fpagnoux) [inline CHANGELOG.md]: J'édite le changelog, pour expliciter que le modèle était compatible avec Python3, mais pas certains scripts additionnels.

**Comment by fpagnoux** [comment]
> Ok, the tests were passing because in circleCI it's only specifically running nosetests, while in the Makefile it's also compiling all the python files to check for syntax errors.

> I also realized that the check-no-prints, check-syntax-errors and flake8 are not run at all in circleCI. I've opened # for that matter.

The France model was already compatible with Python 3, but true, many additional scripts were not, thanks for dealing with that!

 The makefile is a mess, and no one was brave enough to flake OpenFisca France after years of laxness.

**Comment by fpagnoux** [comment]
Rebased, waiting for the tests to pass to merge

**Comment by magopian** [comment]
@vinyll @fpagnoux rebased!

**Pull request merged**